File: Cap 04/4.6 - Operadores Logicos/4.6.3 - Operador Logico de Negacao/nottest.cs
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.6.3
 * 
 * Programa ..: OPERACAO DE NEGACAO ".NAO. / NOT"
*/

using System;

namespace TesteLogico
{
  class Program
  {
	  
    static int A, B, X, C;
    static void Main(string[] args)
    {
		
      Console.WriteLine("TESTE LOGICO OPERADOR: NOT\n");
      
      Console.Write("Entre valor inteiro <A>: ");
      A = Convert.ToInt32(Console.ReadLine());
      
      Console.Write("Entre valor inteiro <B>: ");
      B = Convert.ToInt32(Console.ReadLine());
      
      Console.Write("Entre valor inteiro <X>: ");
      X = Convert.ToInt32(Console.ReadLine());
      
      if (!(X > 5))
      {
        C = A + B;
      }
      else
      {
        C = A - B;
      }
      
      Console.WriteLine("O valor de C = " + C);
      
    }
    
  }
  
}

File: Cap 04/4.6 - Operadores Logicos/4.6.3 - Operador Logico de Negacao/nottest.go
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.6.3
 * 
 * Programa ..: OPERACAO DE NEGACAO ".NAO. / NOT"
 */

package main

import (
  "fmt"
)

var (
  A, B, X, C int
)

func main() {
	
  fmt.Println("TESTE LOGICO OPERADOR: NOT\n")
  
  fmt.Print("Entre valor inteiro <A>: ")
  fmt.Scan(&A)
  
  fmt.Print("Entre valor inteiro <B>: ")
  fmt.Scan(&B)
  
  fmt.Print("Entre valor inteiro <X>: ")
  fmt.Scan(&X)
  
  if !(X > 5) {
    C = A + B
  } else {
    C = A - B
  }
  
  fmt.Println("O valor de C =", C)
}
